g/go-sockaddr/go-sockaddr_v1.0.2_ubi8.4.sh

```diff
@@ -0,0 +1,62 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: go-sockaddr
+# Version	: v1.0.2
+# Source repo	: github.com/hashicorp/go-sockaddr
+# Tested on	: UBI 8.4
+# Language      : GO
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: BulkPackageSearch Automation {maintainer}
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e
+
+PACKAGE_NAME=go-sockaddr
+PACKAGE_VERSION=${1:-v1.0.2}
+PACKAGE_URL=github.com/hashicorp/go-sockaddr
+
+yum install -y git wget gcc-c++
```

Comment:
I see tests are failing on fresh container, below are the logs :

```
?       github.com/hashicorp/go-sockaddr/cmd/sockaddr/command   [no test files]
--- FAIL: TestSockAddr_Parse (0.00s)
    --- FAIL: TestSockAddr_Parse/math_network (0.00s)
        template_test.go:274: "math network": Expected "127.0.0.2 ::1 fe80::2", received ""
            "{{GetAllInterfaces | include \"name\" \"^lo0$\" | include \"type\" \"IP\" | math \"network\" \"+2\" | sort \"+type,+address\" | join \"address\" \" \" }}"
    --- FAIL: TestSockAddr_Parse/sort_asc_address#01 (0.01s)
        template_test.go:274: "sort asc address": Expected "::1 fe80::1/64 ", received ""
            "{{with $ifSet := include \"name\" \"lo0\" . }}{{ range include \"type\" \"IPv6\" $ifSet | sort \"address\"}}{{ .SockAddr }} {{end}}{{end}}"
    --- FAIL: TestSockAddr_Parse/math_address_+_overflow+wrap (0.01s)
        template_test.go:274: "math address + overflow+wrap": Expected "126.255.255.255 ::ffff:ffff fe80::ffff:ffff", received ""
            "{{GetAllInterfaces | include \"name\" \"^lo0$\" | include \"type\" \"IP\" | math \"address\" \"+4294967294\" | sort \"+type,+address\" | join \"address\" \" \" }}"
    --- FAIL: TestSockAddr_Parse/math_network_+_wrap (0.01s)
        template_test.go:274: "math network + wrap": Expected "127.0.255.8 ::1 fe80::ff00:ff08", received ""
            "{{GetAllInterfaces | include \"name\" \"^lo0$\" | include \"type\" \"IP\" | math \"network\" \"+4278255368\" | sort \"+type,+address\" | join \"address\" \" \" }}"
    --- FAIL: TestSockAddr_Parse/math_address_- (0.01s)
        template_test.go:274: "math address -": Expected "126.255.255.1 fe7f:ffff:ffff:ffff:ffff:ffff:ffff:ff01 ffff:ffff:ffff:ffff:ffff:ffff:ffff:ff01", received ""
            "{{GetAllInterfaces | include \"name\" \"^lo0$\" | include \"type\" \"IP\" | math \"address\" \"-256\" | sort \"+type,+address\" | join \"address\" \" \" }}"
    --- FAIL: TestSockAddr_Parse/GetPrivateIPs (0.01s)
        template_test.go:274: "GetPrivateIPs": Expected "10.1.2.3 fe80::1025:f732:1001:203", received "172.18.0.1 10.88.0.1 172.17.0.1 10.10.81.76 10.100.81.76 10.200.81.76"
            "{{GetPrivateIPs}}"
    --- FAIL: TestSockAddr_Parse/GetInterfaceIPs (0.01s)
        template_test.go:274: "GetInterfaceIPs": Expected "10.1.2.3 and 172.16.4.6", received ""
            "{{GetInterfaceIPs \"en0\"}}"
    --- FAIL: TestSockAddr_Parse/sort_desc_address#01 (0.00s)
        template_test.go:274: "sort desc address": Expected "fe80::1 ::1", received ""
            "{{ . | include \"name\" \"lo0\" | include \"type\" \"IPv6\" | sort \"type,-address\" | join \"address\" \" \" }}"
    --- FAIL: TestSockAddr_Parse/with_variable_pipeline (0.01s)
        template_test.go:274: "with variable pipeline": Expected "lo0 127.0.0.1/8 ", received ""
            "{{with $ifSet := include \"type\" \"IPv4\" . | include \"name\" \"^lo0$\"}}{{range $ifSet }}{{.Name}} {{end}}{{range $ifSet}}{{.SockAddr}} {{end}}{{end}}"
    --- FAIL: TestSockAddr_Parse/include_"rfc" (0.00s)
        template_test.go:274: "include \"rfc\"": Expected "en0", received "enp0s9"
            "{{(. | include \"rfc\" \"1918\" | attr \"name\")}}"
    --- FAIL: TestSockAddr_Parse/GetPublicIPs (0.01s)
        template_test.go:274: "GetPublicIPs": Expected "1.2.3.4 6.7.8.9", received "129.40.81.76"
            "{{GetPublicIPs}}"
    --- FAIL: TestSockAddr_Parse/sort_desc_address (0.00s)
        template_test.go:274: "sort desc address": Expected "127.0.0.1 fe80::1 ::1", received ""
            "{{ . | include \"name\" \"lo0\" | sort \"type,-address\" | join \"address\" \" \" }}"
    --- FAIL: TestSockAddr_Parse/basic_include_"name" (0.02s)
        template_test.go:274: "basic include \"name\"": Expected "[127.0.0.1/8 {1 16384 lo0  up|loopback|multicast} ::1 {1 16384 lo0  up|loopback|multicast} fe80::1/64 {1 16384 lo0  up|loopback|multicast}]", received "[]"
            "{{GetAllInterfaces | include \"name\" \"lo0\" | printf \"%v\"}}"
    --- FAIL: TestSockAddr_Parse/sort_asc_address (0.00s)
        template_test.go:274: "sort asc address": Expected "127.0.0.1 ::1 fe80::1", received ""
            "{{ . | include \"name\" \"lo0\" | sort \"type,address\" | join \"address\" \" \" }}"
    --- FAIL: TestSockAddr_Parse/math_network_-_underflow+wrap (0.01s)
        template_test.go:274: "math network - underflow+wrap": Expected "127.255.255.248 ::1 fe80::ffff:ffff:ff:fff8", received ""
            "{{GetAllInterfaces | include \"name\" \"^lo0$\" | include \"type\" \"IP\" | sort \"+type,+address\" | math \"network\" \"-4278190088\" | join \"address\" \" \" }}"
    --- FAIL: TestSockAddr_Parse/math_network_- (0.02s)
        template_test.go:274: "math network -": Expected "127.255.255.254 ::1 fe80::ffff:ffff:ffff:fffe", received ""
            "{{GetAllInterfaces | include \"name\" \"^lo0$\" | include \"type\" \"IP\" | math \"network\" \"-2\" | sort \"+type,+address\" | join \"address\" \" \" }}"
    --- FAIL: TestSockAddr_Parse/range_sample_on_lo0 (0.01s)
        template_test.go:274: "range sample on lo0": Expected "lo0/127.0.0.1 lo0/::1 lo0/fe80::1 ", received ""
            "{{with $ifAddrs := . | exclude \"rfc\" \"1918\" | include \"name\" \"lo0\" | sort \"type,address\" }}{{range $ifAddrs }}{{.Name}}/{{.SockAddr.NetIP}} {{end}}{{end}}"
    --- FAIL: TestSockAddr_Parse/non-RFC1918_on_on_lo0 (0.01s)
        template_test.go:274: "non-RFC1918 on on lo0": Expected "true", received "false"
            "{{. | exclude \"rfc\" \"1918\" | include \"name\" \"lo0\" | sort \"type,address\" | len | eq 3}}"
    --- FAIL: TestSockAddr_Parse/include_"type"_"IPv6 (0.00s)
        template_test.go:274: "include \"type\" \"IPv6": Expected "[::1 {1 16384 lo0  up|loopback|multicast} fe80::1/64 {1 16384 lo0  up|loopback|multicast}]", received "[]"
            "{{. | include \"type\" \"IPv6\" | include \"name\" \"^lo0$\" | sort \"address\" }}"
    --- FAIL: TestSockAddr_Parse/sort_asc_address_old (0.01s)
        template_test.go:274: "sort asc address old": Expected "127.0.0.1/8 ::1 fe80::1/64 ", received ""
            "{{with $ifSet := include \"name\" \"lo0\" . }}{{ range include \"type\" \"IPv4\" $ifSet | sort \"address\"}}{{ .SockAddr }} {{end}}{{ range include \"type\" \"IPv6\" $ifSet | sort \"address\"}}{{ .SockAddr }} {{end}}{{end}}"
    --- FAIL: TestSockAddr_Parse/exclude_"type" (0.01s)
        template_test.go:274: "exclude \"type\"": Expected "lo0 ::1 fe80::1/64 ", received " "
            "{{. | exclude \"type\" \"IPv4\" | include \"name\" \"^lo0$\" | sort \"address\" | unique \"name\" | join \"name\" \" \"}} {{range . | exclude \"type\" \"IPv4\" | include \"name\" \"^lo0$\"}}{{.SockAddr}} {{end}}"
    --- FAIL: TestSockAddr_Parse/range_"dot"_example (0.00s)
        template_test.go:274: "range \"dot\" example": Expected "lo0 127.0.0.1/8 lo0 ::1 lo0 fe80::1/64 ", received ""
            "{{range . | include \"type\" \"IP\" | include \"name\" \"^lo0$\"}}{{.Name}} {{.SockAddr}} {{end}}"
    --- FAIL: TestSockAddr_Parse/math_address_+_overflow (0.01s)
        template_test.go:274: "math address + overflow": Expected "|128.0.0.2 -- 128.0.0.0/16777216|::100:2 -- ::100:2/1|fe80::100:2 -- fe80::/18446744073709551616|", received "|"
            "|{{- with $ifAddrs := GetAllInterfaces | include \"name\" \"^lo0$\" | include \"type\" \"IP\" | math \"address\" \"+16777217\" | sort \"+type,+address\" -}}\n  {{- range $ifAddrs -}}\n    {{- attr \"address\" . }} -- {{ attr \"network\" . }}/{{ attr \"size\" . }}|{{ end -}}\n{{- end -}}"
    --- FAIL: TestSockAddr_Parse/math_address_+ (0.00s)
        template_test.go:274: "math address +": Expected "127.0.0.3 ::3 fe80::3", received ""
            "{{GetAllInterfaces | include \"name\" \"^lo0$\" | include \"type\" \"IP\" | math \"address\" \"+2\" | sort \"+type,+address\" | join \"address\" \" \" }}"
    --- FAIL: TestSockAddr_Parse/lo0_flags_up_and_limit_1 (0.01s)
        template_test.go:274: "lo0 flags up and limit 1": Expected "::1", received ""
            "{{. | include \"name\" \"lo0\" | include \"flag\" \"up\" | sort \"-type,+address\" | attr \"address\" }}"
    --- FAIL: TestSockAddr_Parse/math_address_-_underflow (0.01s)
        template_test.go:274: "math address - underflow": Expected "127.255.255.255 fe7f:ffff:ffff:ffff:ffff:ffff:ff:ffff ffff:ffff:ffff:ffff:ffff:ffff:ff:ffff", received ""
            "{{GetAllInterfaces | include \"name\" \"^lo0$\" | include \"type\" \"IP\" | math \"address\" \"-4278190082\" | sort \"+type,+address\" | join \"address\" \" \" }}"
    --- FAIL: TestSockAddr_Parse/ifAddrs1 (0.00s)
        template_test.go:274: "ifAddrs1": Expected "[127.0.0.1/8 {1 16384 lo0  up|loopback|multicast}]", received "[]"
            "{{. | include \"type\" \"IPv4\" | include \"name\" \"^lo0$\"}}"
    --- FAIL: TestSockAddr_Parse/exclude_"name" (0.00s)
        template_test.go:274: "exclude \"name\"": Expected "127.0.0.1 ::1 fe80::1", received ""
            "{{. | include \"name\" \"^(en|lo)0$\" | exclude \"name\" \"^en0$\" | sort \"type\" | sort \"address\" | join \"address\" \" \" }}"
    --- FAIL: TestSockAddr_Parse/join_address (0.00s)
        template_test.go:274: "join address": Expected "::1 fe80::1", received ""
            "{{. | include \"name\" \"lo0\" | include \"type\" \"IPv6\" | sort \"address\" | join \"address\" \" \" }}"
    --- FAIL: TestSockAddr_Parse/"dot"_pipeline,_IPv4_type (0.01s)
        template_test.go:274: "\"dot\" pipeline, IPv4 type": Expected "[127.0.0.1/8 {1 16384 lo0  up|loopback|multicast}]", received "[]"
            "{{. | include \"type\" \"IPv4\" | include \"name\" \"^lo0$\" | sort \"type\" | sort \"address\" }}"
    --- FAIL: TestSockAddr_Parse/ifAddrs2 (0.01s)
        template_test.go:274: "ifAddrs2": Expected "[127.0.0.1/8 {1 16384 lo0  up|loopback|multicast} ::1 {1 16384 lo0  up|loopback|multicast} fe80::1/64 {1 16384 lo0  up|loopback|multicast}]", received "[]"
            "{{. | include \"type\" \"IP\" | include \"name\" \"^lo0$\" | sort \"type\" | sort \"address\" }}"
    --- FAIL: TestSockAddr_Parse/HashiCorpDefault2016 (0.01s)
        template_test.go:270: "HashiCorpDefault2016": bad: unable to execute sockaddr input "\n{{- with $addr := GetAllInterfaces | include \"type\" \"IP\" | include \"rfc\" \"1918|6598\" | sort \"address\" | attr \"address\" -}}\n\n  {{- if ($addr | len) gt 0 -}}\n    {{- print \"true\" -}}{{/* print $addr*/ -}}\n  {{- end -}}\n{{- end -}}": template: sockaddr.Parse:4:9: executing "sockaddr.Parse" at <($addr | len) gt 0>: can't give argument to non-function $addr | len
    --- FAIL: TestSockAddr_Parse/join_name (0.01s)
        template_test.go:274: "join name": Expected "lo0 lo0", received ""
            "{{. | include \"name\" \"lo0\" | include \"type\" \"IPv6\" | sort \"address\" | join \"name\" \" \" }}"
    --- FAIL: TestSockAddr_Parse/better_example_for_IP_types (0.00s)
        template_test.go:274: "better example for IP types": Expected "[127.0.0.1/8 {1 16384 lo0  up|loopback|multicast} ::1 {1 16384 lo0  up|loopback|multicast} fe80::1/64 {1 16384 lo0  up|loopback|multicast}]", received "[]"
            "{{. | include \"type\" \"IPv4|IPv6\" | include \"name\" \"^lo0$\" | sort \"type\" | sort \"address\" }}"
    --- FAIL: TestSockAddr_Parse/GetDefaultInterface (0.01s)
        template_test.go:274: "GetDefaultInterface": Expected "en0", received "enp0s1"
            "{{GetDefaultInterfaces | include \"type\" \"IPv4\" | attr \"name\" }}"
    --- FAIL: TestSockAddr_Parse/include_"name"_regexp (0.01s)
        template_test.go:274: "include \"name\" regexp": Expected "127.0.0.1 ::1 fe80::1", received ""
            "{{GetAllInterfaces | include \"name\" \"^(en|lo)0$\" | exclude \"name\" \"^en0$\" | sort \"type\" | sort \"address\" | join \"address\" \" \" }}"
    --- FAIL: TestSockAddr_Parse/lo0_limit_1 (0.01s)
        template_test.go:274: "lo0 limit 1": Expected "1", received "0"
            "{{. | include \"name\" \"lo0\" | include \"type\" \"IPv6\" | sort \"address\" | limit 1 | len}}"
FAIL
FAIL    github.com/hashicorp/go-sockaddr/template       0.049s
FAIL
```

Reply:
I had missed to add README file. The failing tests are in parity with intel x86.
Added README file.